sunpy/net/jsoc/jsoc.py

```diff
@@ -58,7 +63,10 @@ def __len__(self):
             return len(self.table)
 
     def __getitem__(self, item):
-        return type(self)(self.table[item])
+        resp = type(self)(self.table[item])
```

Comment:
The pattern I went with in #3770 (although not in the `JSOCClient` apparently) was this https://github.com/sunpy/sunpy/pull/3770/files#diff-0c4a0061c4c5eefc3ba432fa7a2b1280R157-R160

Would this apply here?

Reply:
but JSOCResponse takes only `table` as an argument. Should I add `**kwargs` as well?

Reply:
I was referring to indexing like `[0:1]` rather than `[0]` so you get an iterable object back.

Reply:
oh okay